ui/app/pages/create-account/connect-hardware/index.js

```diff
@@ -171,12 +174,28 @@ class ConnectHardwareForm extends Component {
   }
 
   renderError () {
+    if (this.state.error === 'U2F') {
+      return (
+        <p
+          className="hw-connect__error"
+        >
+          {this.context.t('troubleConnectingToWallet', [this.state.device])}
+          <a
+            href="https://metamask.zendesk.com/hc/en-us/articles/360020394612-How-to-connect-a-Trezor-or-Ledger-Hardware-Wallet"
+            target="_blank"
+            rel="noopener noreferrer"
+            className="hw-connect__link"
+            style={{ marginLeft: '5px', marginRight: '5px' }}
+          >
+            {this.context.t('walletConnectionGuide')}
+          </a>
+          {this.context.t('andTryAgain')}
+        </p>
```

Comment:
We should be able to merge these into a single translatable message, passing in JSX for the substitutions where needed:

```jsx
<p className="hw-connect__error">
  {this.context.t('troubleConnectingToWallet', [this.state.device, (
    <a
      href="https://metamask.zendesk.com/hc/en-us/articles/360020394612-How-to-connect-a-Trezor-or-Ledger-Hardware-Wallet"
      target="_blank"
      rel="noopener noreferrer"
      className="hw-connect__link"
      style={{ marginLeft: '5px', marginRight: '5px' }}
     >
       {this.context.t('walletConnectionGuide')}
     </a>
   )])}
</p>
```

With the combined message something like: `We had trouble connecting to your $1, try reviewing $2 and try again.`